src/api/controllers/misc.ts

```diff
@@ -82,10 +82,10 @@ export async function handleInternalUserUpdate(data: DiscordPayload) {
 }
 
 /** This function is the internal handler for the voice state update event. Users can override this with controllers if desired. */
-export async function handleInternalVoiceStateUpdate(data: DiscordPayload) {
+export async function handleInternalVoiceStateUpdate(data: GatewayPayload) {
   if (data.t !== "VOICE_STATE_UPDATE") return;
 
-  const payload = data.d as VoiceStateUpdatePayload;
+  const payload = data.d as VoiceStatePayload;
```

Comment:
Rename `VoiceStatePayload` to `VoiceStateUpdateEventPayload`?

Reply:
Thats not a type from gateway.ts.

Reply:
It is a type from voice.ts the gateway event uses this here instead of a custom one.